bin/sdk/gutenberg.js

```diff
@@ -7,8 +7,13 @@ const path = require( 'path' );
 exports.config = ( {
 	argv: { editorScript, viewScript, outputDir, outputEditorFile, outputViewFile },
 	getBaseConfig,
+	__rootDir,
 } ) => {
-	const baseConfig = getBaseConfig( { externalizeWordPressPackages: true } );
+	const baseConfig = getBaseConfig( {
+		externalizeWordPressPackages: true,
+		stylesNamespacing: 'calypso',
+		stylesNamespacingExclude: path.join( __rootDir, 'client', 'gutenberg' ),
```

Comment:
do we need a customizable exclude list? do you think we could get by with a whitelist instead just namespacing Calypso's `components` directory and maybe some others?

Reply:
You had the right intuition — it was indeed possible to simplify this and get rid of the custom exclusion-list. Thanks for pointing it out!